client/src/components/Profile/style.css

```diff
@@ -1,6 +1,11 @@
 .alignLeft {
     text-align: left;
-    color: white;
+    color: rgb(10, 10, 10);
+}
+
+.bioText{
```

Comment:
Once again I believe the text for bio should be white, and regardless it should be unified with the rest of text on profile page

Reply:
done